### PR TITLE
Support translation of uniform group extension instructions form SPIRV to LLVM

### PR DESCRIPTION
**Pull request by wwXing0**

### CODE REVIEW

**Comment by MrSidims** [comment]
@wwXing0 thanks! Could you please add a test for the case? @KornevNikita please assist in the test writing if necessary.

**Comment by wwXing0** [comment]
@MrSidims , please help to running workflows and review, Thanks

File: test/transcoding/SPV_KHR_uniform_group_instructions/group-instructions.ll [hunk 1]
```diff
@@ -32,14 +32,17 @@
...
 ; CHECK-SPIRV: GroupFMulKHR [[#TypeFloat]] [[#]] [[#Scope]] 0 [[#Val2]]
 
-; CHECK-LLVM: %call1 = call spir_func i32 @_Z26__spirv_GroupBitwiseAndKHR{{.*}}(i32 2, i32 0, i32 0)
-; CHECK-LLVM: %call2 = call spir_func i32 @_Z25__spirv_GroupBitwiseOrKHR{{.*}}(i32 2, i32 0, i32 0)
-; CHECK-LLVM: %call3 = call spir_func i32 @_Z26__spirv_GroupBitwiseXorKHR{{.*}}(i32 2, i32 0, i32 0)
-; CHECK-LLVM: %call4 = call spir_func i1 @_Z26__spirv_GroupLogicalAndKHR{{.*}}(i32 2, i32 0, i1 false)
-; CHECK-LLVM: %call5 = call spir_func i1 @_Z25__spirv_GroupLogicalOrKHR{{.*}}(i32 2, i32 0, i1 false)
-; CHECK-LLVM: %call6 = call spir_func i1 @_Z26__spirv_GroupLogicalXorKHR{{.*}}(i32 2, i32 0, i1 false)
-; CHECK-LLVM: %call7 = call spir_func i32 @_Z20__spirv_GroupIMulKHR{{.*}}(i32 2, i32 0, i32 0)
-; CHECK-LLVM: %call8 = call spir_func half @_Z20__spirv_GroupFMulKHR{{.*}}(i32 2, i32 0, half 0xH0000)
```
Comment (MrSidims) [inline test/transcoding/SPV_KHR_uniform_group_instructions/group-instructions.ll:42]: This checks should remain but under --check-prefix=CHECK-LLVM-SPIRV, please see https://github.com/KhronosGroup/SPIRV-LLVM-Translator/blob/master/test/transcoding/spirv-types.ll lines 10-15

Comment (MrSidims) [reply test/transcoding/SPV_KHR_uniform_group_instructions/group-instructions.ll:42]: Updated the test

**Review by MrSidims — APPROVED** [review]

File: test/transcoding/SPV_KHR_uniform_group_instructions/group-instructions.ll (hunk 1 above)
Comment (wwXing0) [reply test/transcoding/SPV_KHR_uniform_group_instructions/group-instructions.ll:42]: Thanks for helping.

**Pull request merged**